### DISCUSSION TITLE
About building Milvus cluster

### DISCUSSION THREAD

**Discussion by mybabyiscue** [milvus-io/milvus#27530] [Q&A and General discussion]
Since I am new to Milvus, I want to build a two-node cluster. However, there is no detailed installation tutorial on the official website or Google. Could any boss please provide a relevant document? Thank you so much

**Comment by yhmo** [comment]
The recommended way to deploy milvus cluster is: 
[helm](https://milvus.io/docs/install_cluster-helm.md) or [operator](https://milvus.io/docs/install_cluster-milvusoperator.md)

**Reply by mybabyiscue** [reply]
After consulting online information, I found that storageclass needs to be installed and configured. Is this step necessary to install it? Are there any detailed installation instructions? I didn't find relevant information on github. Is it convenient for you to provide

**Reply by LoveEachDay** [reply]
Milvus installation relies on default storageclass to persistence data. You can contact your kubernetes administrator to set up a default storageclass. 

You can use `openebs` or rancher `local-path` to use your local storage.
[1] https://openebs.io/docs
[2] https://github.com/rancher/local-path-provisioner
